components/bl-cropper-component/styles/index.less

```diff
@@ -0,0 +1,139 @@
+@bl-customComponent-cropper-button-borderRadius: 4px;
+@bl-customComponent-cropper-button-margin: 2px;
+@bl-customComponent-cropper-button-padding: 6px 16px;
+@bl-customComponent-cropper-button-background: @bl-customComponent-cropper-themeColor;
+@bl-customComponent-cropper-button-color: contrast(@bl-customComponent-cropper-button-background);
+@bl-customComponent-cropper-button-shadowColor: @bl-customComponent-cropper-shadowColor;
+@bl-customComponent-cropper-button-shadowHover: 0px 2px 4px -1px fade(@bl-customComponent-cropper-button-shadowColor, 20%), 0px 4px 5px 0px fade(@bl-customComponent-cropper-button-shadowColor, 14%), 0px 1px 10px 0px fade(@bl-customComponent-cropper-button-shadowColor, 12%);
+@bl-customComponent-cropper-button-shadowDisabled: none;
+@bl-customComponent-cropper-button-disabledBackground: @bl-customComponent-cropper-disabledColor;
+@bl-customComponent-cropper-button-disabledColor: contrast(@bl-customComponent-cropper-button-disabledBackground);
+
+@bl-customComponent-cropper-icon-fontSize: 20px;
+@bl-customComponent-cropper-icon-margin: 0;
+@bl-customComponent-cropper-icon-padding: 4.4px 16px;
+
+@bl-customComponent-cropper-dropImageArea-width: 100%;
+@bl-customComponent-cropper-dropImageArea-height: 100%;
+
+@bl-customComponent-cropper-browseButton-width: 100%;
+@bl-customComponent-cropper-browseButton-height: 100%;
+@bl-customComponent-cropper-browseButton-color: @bl-customComponent-cropper-themeColor;
+
+.bl-customComponent-cropper {
+  display: flex;
+  flex-direction: column;
+  gap: @bl-customComponent-cropper-gap;
+  width: @bl-customComponent-cropper-width;
+  padding: @bl-customComponent-cropper-padding;
+  border: @bl-customComponent-cropper-border;
+  border-radius: @bl-customComponent-cropper-borderRadius;
+
+  .header-toolbar {
+    display: flex;
+    justify-content: space-between;
+
+    .buttons-wrapper {
+      display: flex;
+
+      input {
+        display: none;
+      }
+    }
+  }
+
+  .container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    .drop-image-area {
+      width: @bl-customComponent-cropper-dropImageArea-width;
+      height: @bl-customComponent-cropper-dropImageArea-height;
+      
+      label {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        color: @bl-customComponent-cropper-browseButton-color;
+        width: @bl-customComponent-cropper-browseButton-width;
+        height: @bl-customComponent-cropper-browseButton-height;
+        cursor: pointer;
+  
+        &:hover {
+          text-decoration: underline;
+        }
+  
+        input {
+          display: none;
+        }
+      }
+    }
+
+    img {
+      display: block;
+      max-width: @bl-customComponent-cropper-maxWidth;
+      max-height: @bl-customComponent-cropper-maxHeight;
+    }
+  }
+
+  .footer-toolbar {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    align-items: center;
+
+    .buttons-group {
```

Comment:
I will offer this variant of button group styling
![image](https://user-images.githubusercontent.com/15347646/201121028-7bde887d-21e0-4e21-af84-43a8cd508cb9.png)

Reply:
![Screenshot_12](https://user-images.githubusercontent.com/93279075/201299776-db25b688-a928-4aba-b315-9970eed856a9.png)